index.html

```diff
@@ -160,7 +160,46 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
             <a href="https://github.com/w3c/mediacapture-region/issues/18">issue #18</a>.
           </p>
         </div>
-        <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+        <dl data-link-for="CropTarget" data-dfn-for="CropTarget">
+          <dt>
+            <dfn>fromElement()</dfn>
+          </dt>
+          <dd>
+            <p>
+              Calling {{CropTarget/fromElement}} on an {{Element}} of a supported type associates
```

Comment:
With an element

Reply:
I'd also remove "supported type" as that seems vague, but I appreciate the code move is causing the diff to grow, and would happily deal with that in a followup as well.

Reply:
Done.